### PR TITLE
Add meeting attendees (required and optionals) and is_recurring

### PR DESCRIPTION
**Pull request by mguyard**
Please find my PR to add attendees and isRecurring

I've also a question about "whole_day". This is something available in "ev.is_all_day", do you have encountering some bugs with this to use a custom function ?
https://github.com/pschmitt/jcalapi/blob/757c8040403bfefa3ff85c7c4366152942b0c8eb/jcalapi/backend/exchange.py#L165

### CODE REVIEW

File: jcalapi/backend/exchange.py [hunk 1]
```diff
@@ -208,18 +208,29 @@ def sync_get_exchange_events(
...
                 "organizer": ev.organizer.name,
+                "attendees": ev_attendees,
                 "summary": ev.subject,
                 "description": ev_body,
                 "body": ev.body,
                 "location": location,
                 "start": ev_start,
                 "end": ev_end,
                 "whole_day": whole_day,
+                "isRecurring": ev.is_recurring,
```
Comment (pschmitt) [inline jcalapi/backend/exchange.py:233]:
```suggestion
                "is_recurring": ev.is_recurring,
```

**Comment by pschmitt** [comment]
Thanks!

> do you have encountering some bugs with this to use a custom function ?

I must admit that I cannot remember, but yeah - probably. That field is hard to miss after all.

**Comment by pschmitt** [comment]
Please GPG sign your commits :)

**Comment by mguyard** [comment]
Thanks

**Pull request merged**